code/modules/urist/antagonist/conspiracies.dm

```diff
@@ -0,0 +1,105 @@
+//This file defines the various conspiracies
+//Unfortunately large amounts of boilerplate code, but cleaning it would require some major work
+//Begin terrible SS13-pun names, credit/blame goes to some ancient /tg/station thread:
+
+//////////////////////////BUILDABORGS///////////////////////////////
+
+var/datum/antagonist/agent/buildaborg/buildaborgs
+/datum/antagonist/agent/buildaborg/New()
+	..()
+	buildaborgs = src
+
+/datum/antagonist/agent/buildaborg
+	id = "Buildaborg"
+	role_text = "Buildaborg Group Leader"
+	role_text_plural = "Buildaborg Agents"
+	faction_role_text = "Buildaborg Agent"
+	faction_descriptor = "Buildaborg Group"
+	faction_verb = /mob/living/proc/convert_to_conspiracy
+	faction_welcome = "Help the cause. Do not harm your fellow agents."
+
+/mob/living/proc/convert_to_buildaborg(mob/M as mob in oview(src))
```

Comment:
it would totally be possible to condense all the convert procs into one single proc. Which would be a lot nicer and easier to work with.